graphyte.py

```diff
@@ -168,14 +170,15 @@ def _thread_loop(self):
             current_time = time.time()
             if current_time - last_check_time >= self.interval:
                 last_check_time = current_time
-                if messages:
-                    self.send_socket(b''.join(messages))
-                    messages = []
+                for i in range(0, len(messages), self.batch_size):
+                        batch = messages[i:i + self.batch_size]
```

Comment:
Sorry for the nitting, but let's just fix the indentation here (the body of the for loop is indented too far).

Reply:
After that tweak I'll merge!